### PR TITLE
Add Service Bouncer Project

### PR DESCRIPTION
**Pull request by BlythMeister**

### CODE REVIEW

**Comment by shiftkey** [comment]
@BlythMeister thanks!

**Comment by ritwik12** [comment]
The site is not working as it is spelled wrong. Please correct it.

File: _data/projects/servicebouncer.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: ServiceBouncer
+desc: Service Manager - Microservices meets DevOps
+site: https://github.com/PaulStovell/ServiceBounce
```
Comment (shiftkey) [inline _data/projects/servicebouncer.yml:3]: Whoops, missed the `r` here

**Comment by ritwik12** [comment]
Yeah 😅, that's why I put this PR on  hold

**Pull request merged**